pages/implementations/mapping.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Mapping To Rule
+---
+
+## Implementation Scoping
+
+An implementation does not need to have a one-to-one mapping to ACT-R rules. An implementation can test multiple ACT-R rules in a single check. In manual test procedures, it is common to test multiple things at once. For instance, a test procedure looking at the language of a page will simultaniously test whether or not the `<html>` element has a `lang` attribute, if the value of the `lang` attribute is valid, and if the value matches the language of the page. This is 3 separate rules, covered in a single test procedure of an implementation.
```

Comment:
`ACT-R rules` > `ACT Rules`

Reply:
Disagree. ACT-R rules are rules written by the ACT-R community group. ACT rules are rules conforming to the ACT Rules Format.